app/assets/javascripts/alchemy/alchemy.sitemap.js.coffee

```diff
@@ -4,12 +4,51 @@ window.Alchemy = {} if typeof(window.Alchemy) is 'undefined'
 Alchemy.Sitemap =
 
   # Storing some objects.
-  init: ->
+  init: (url, page_root_id, full, callback) ->
```

Comment:
As discussed below we would enhance readability if we use an `options` object here, instead of anonymous arguments.